app/Descriptors.php

```diff
@@ -29,6 +29,22 @@ private function validateSelectType($value): bool
         return $allowedValues->contains($value);
     }
 
+    /**
+     * This is the validator for the Multi-Select value type.
+     */
+    private function validateMultiSelectType($values): bool
+    {
+        $isValid = true;
+        $values = explode(', ', $values);
+        foreach ($values as $selectedValue) {
+            if (!$this->validateSelectType(trim($selectedValue))) {
```

Comment:
This calls a pre-existing validator for the base Select descriptor type in order to minimize code duplication. Upon a single validation error, this method returns a false ("invalid") response.